pandas/tests/indexes/interval/test_interval.py

```diff
@@ -435,6 +435,14 @@ def test_get_loc_value(self):
         idx = IntervalIndex.from_arrays([0, 2], [1, 3])
         pytest.raises(KeyError, idx.get_loc, 1.5)
 
+        # GH25087, test get_loc returns key error for interval indexes
```

Comment:
Can you put this in a new test? (you can leave it in this place, but just put a `def test_get_loc_invalid_key(self)` above this line) 
Reason is that the other test is commented to be replaced, but this new test we want to keep.

Reply:
Can you do this one?

Reply:
Sure.

Reply:
Have got a few broken tests to fix. Hoping to make the build green in the coming days.